types/nodejs-captcha/index.d.ts

```diff
@@ -0,0 +1,68 @@
+// Type definitions for ssdeep.js 0.0
```

Comment:
This seems to be the wrong package name

```suggestion
// Type definitions for nodejs-captcha 0.0
```